app/main.py

```diff
@@ -2,0 +1,25 @@
+from app.cafe import Cafe
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotVaccinatedError:
+            return "All friends should be vaccinated"
+        except OutdatedVaccineError:
+            return "All friends should be vaccinated"
```

Comment:
The error messages in these exceptions are the same, which might be misleading. It's better to provide more specific messages for different exceptions. For example, for 'NotVaccinatedError' you can return 'All friends should be vaccinated', and for 'OutdatedVaccineError' - 'All friends should have up-to-date vaccines'.